package-lock.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "copy",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "copy",
-      "version": "1.0.0",
+      "version": "1.1.0",
       "devDependencies": {
         "@types/node": "20.9.0",
         "@types/vscode": "^1.78.0",
-        "@typescript-eslint/eslint-plugin": "^6.7.3",
-        "@typescript-eslint/parser": "^6.7.3",
-        "@vscode/test-electron": "^2.3.4",
-        "eslint": "^8.50.0",
+        "@typescript-eslint/eslint-plugin": "^6.10.0",
```

Comment:
and do not change anything in package.json

Reply:
@Prabhakar-Poudel if you check the diff, you can see that package.json is not at all touched, its the package-lock.json autogenerated by npm, and to be clear the version right now is the right version from your package.json.

For some reason the package-lock.json wasnt updated the last time you commited and hence the version is now udpated, If you check the package.json you can see the version is 1.1.0  (commited by you)